### ISSUE TITLE
Supress tensorflow warnings 

### ISSUE THREAD

**Issue by kannandreams** [ritwik12/Celestial-bodies-detection#43]
```
$ python label_image.py test_data/uranus_1.jpg
WARNING:tensorflow:From label_image.py:10: FastGFile.__init__ (from tensorflow.python.platform.gfile) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.gfile.GFile.
WARNING:tensorflow:From label_image.py:13: The name tf.gfile.GFile is deprecated. Please use tf.io.gfile.GFile instead.

WARNING:tensorflow:From label_image.py:17: The name tf.GraphDef is deprecated. Please use tf.compat.v1.GraphDef instead.

2019-11-08 14:01:32.473545: W tensorflow/core/framework/op_def_util.cc:357] Op BatchNormWithGlobalNormalization is deprecated. It will cease to work in GraphDef version 9. Use tf.nn.batch_normalization().
WARNING:tensorflow:From label_image.py:22: The name tf.Session is deprecated. Please use tf.compat.v1.Session instead.

2019-11-08 14:01:32.664871: I tensorflow/core/platform/cpu_feature_guard.cc:145] This TensorFlow binary is optimized with Intel(R) MKL-DNN to use the following CPU instructions in performance critical operations:  SSE4.1 SSE4.2 AVX AVX2 FMA
To enable them in non-MKL-DNN operations, rebuild TensorFlow with the appropriate compiler flags.
2019-11-08 14:01:32.665248: I tensorflow/core/common_runtime/process_util.cc:115] Creating new thread pool with default inter op setting: 8. Tune using inter_op_parallelism_threads for best performance.

```

**Comment by kannandreams** [comment]
As per the documentation, we will change to **tf.io.gfile.GFile**

After fixing , the output looks like this for prediction . Above warnings are gone. I will try to fix other warnings as well. 

```$ python label_image.py test_data/uranus_1.jpg
WARNING:tensorflow:From label_image.py:17: The name tf.GraphDef is deprecated. Please use tf.compat.v1.GraphDef instead.

2019-11-08 14:12:56.416609: W tensorflow/core/framework/op_def_util.cc:357] Op BatchNormWithGlobalNormalization is deprecated. It will cease to work in GraphDef version 9. Use tf.nn.batch_normalization().
WARNING:tensorflow:From label_image.py:22: The name tf.Session is deprecated. Please use tf.compat.v1.Session instead.

2019-11-08 14:12:56.617430: I tensorflow/core/platform/cpu_feature_guard.cc:145] This TensorFlow binary is optimized with Intel(R) MKL-DNN to use the following CPU instructions in performance critical operations:  SSE4.1 SSE4.2 AVX AVX2 FMA
To enable them in non-MKL-DNN operations, rebuild TensorFlow with the appropriate compiler flags.
2019-11-08 14:12:56.617910: I tensorflow/core/common_runtime/process_util.cc:115] Creating new thread pool with default inter op setting: 8. Tune using inter_op_parallelism_threads for best performance.
```

**Comment by ritwik12** [comment]
Great, please send a PR for the same @kannandreams

**Comment by kannandreams** [comment]
Fixed the other two warnings as well. shall i raise a PR for review ? . 
Kindly let me know if you have any concerns or your thoughts on this.

**Comment by ritwik12** [comment]
Please raise, will review Thanks @kannandreams

**Comment by ritwik12** [comment]
Closed by #44
